runners/direct-java/pom.xml

```diff
@@ -280,20 +280,6 @@
 
     <dependency>
       <groupId>com.google.http-client</groupId>
-      <artifactId>google-http-client</artifactId>
-      <version>${google-clients.version}</version>
-      <exclusions>
-        <!-- Exclude an old version of guava that is being pulled
-             in by a transitive dependency of google-api-client -->
-        <exclusion>
-          <groupId>com.google.guava</groupId>
-          <artifactId>guava-jdk5</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
```

Comment:
This change is because we imported the wrong version of `Throwables` (from API client) and thus removed this dependency.

```
[INFO] --- maven-surefire-plugin:2.18.1:test (runnable-on-service-tests) @ direct-runner ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-dependency-plugin:2.10:analyze-only (default) @ direct-runner ---
[WARNING] Unused declared dependencies found:
[WARNING]    com.google.http-client:google-http-client:jar:1.21.0:compile
[INFO] ------------------------------------------------------------------------
```